<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Severin Weitz | WeWi-Systems</author>
 <class>CreateTicketWidgetClass</class>
 <widget class="QWidget" name="CreateTicketWidgetClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1257</width>
    <height>790</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CreateTicketWidget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>3</number>
     </property>
     <widget class="QWidget" name="page_callerInfo">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="5" column="0" colspan="2">
        <widget class="QLabel" name="lb_costUnit">
         <property name="text">
          <string>Cost Unit</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QComboBox" name="cb_caller1"/>
       </item>
       <item row="3" column="0" colspan="2">
        <widget class="QComboBox" name="cb_caller2"/>
       </item>
       <item row="4" column="0" colspan="2">
        <widget class="QComboBox" name="cb_caller3"/>
       </item>
       <item row="8" column="1">
        <widget class="QPushButton" name="pb_p1_back">
         <property name="text">
          <string>Back</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QPushButton" name="pb_p1_next">
         <property name="text">
          <string>Next</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QLabel" name="lb_caller">
         <property name="text">
          <string>Caller / Reporter</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="6" column="0" colspan="2">
        <widget class="QComboBox" name="cb_costUnit"/>
       </item>
       <item row="0" column="0">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_ticketData">
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="1">
        <widget class="QLabel" name="lb_lineMachineRoom">
         <property name="text">
          <string>Line / Machine / Room</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="3">
        <widget class="QLabel" name="lb_ticketInfo">
         <property name="text">
          <string>Ticket Information</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="3">
        <widget class="QTextBrowser" name="tb_ticketInfo"/>
       </item>
       <item row="4" column="0" colspan="3">
        <widget class="QLabel" name="lb_moreInfo">
         <property name="text">
          <string>More Info</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0" colspan="3">
        <widget class="QTextBrowser" name="tb_moreInfo"/>
       </item>
       <item row="6" column="0">
        <widget class="QPushButton" name="pb_p2_next">
         <property name="text">
          <string>Next</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QPushButton" name="pb_p2_back">
         <property name="text">
          <string>Back</string>
         </property>
        </widget>
       </item>
       <item row="6" column="2">
        <widget class="QPushButton" name="pb_p2_cancel">
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="3">
        <widget class="QComboBox" name="cb_lineMachineRoom"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_technic">
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="1" column="1" colspan="3">
        <widget class="QComboBox" name="cb_technician1"/>
       </item>
       <item row="2" column="1" colspan="2">
        <widget class="QLabel" name="lb_technician2">
         <property name="text">
          <string>Technician 2</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1" colspan="3">
        <widget class="QComboBox" name="cb_technician2"/>
       </item>
       <item row="4" column="1" colspan="2">
        <widget class="QLabel" name="lb_technician3">
         <property name="text">
          <string>Technician 3</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1" colspan="3">
        <widget class="QComboBox" name="cb_technician3"/>
       </item>
       <item row="6" column="0">
        <widget class="QPushButton" name="pb_p3_createTicket">
         <property name="text">
          <string>Create Ticket</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QPushButton" name="pb_p3_back">
         <property name="text">
          <string>Back</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="lb_technician1">
         <property name="text">
          <string>Technician 1</string>
         </property>
        </widget>
       </item>
       <item row="6" column="2">
        <widget class="QPushButton" name="pb_p3_cancel">
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page">
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout" stretch="5,5,5,5,0">
         <item>
          <widget class="QGroupBox" name="gb_caller">
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="title">
            <string>Caller</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_5">
            <item row="1" column="0">
             <widget class="QLabel" name="lb_selectedCaller">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="StepWidget" name="w_caller" native="true"/>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="gb_department">
           <property name="title">
            <string>Department</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_6">
            <item row="0" column="0">
             <widget class="StepWidget" name="w_department" native="true"/>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="lb_selectedDepartment">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="gb_line">
           <property name="title">
            <string>Line</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_7">
            <item row="0" column="0">
             <widget class="StepWidget" name="w_line" native="true"/>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="lb_selectedLine">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="gp_employee">
           <property name="title">
            <string>Employee</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_8">
            <item row="0" column="0">
             <widget class="StepWidget" name="w_employee" native="true"/>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="lb_selectedEmployee">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="gB_controls">
           <property name="title">
            <string>Stepcontrol</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QPushButton" name="pb_reset">
              <property name="text">
               <string>Reset</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pb_stepBack">
              <property name="text">
               <string>Previous Step</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="2,1">
         <property name="rightMargin">
          <number>0</number>
         </property>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QGroupBox" name="gB_serach">
             <property name="title">
              <string>Search</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <widget class="QLineEdit" name="le_search"/>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="gB_searchResult">
             <property name="title">
              <string>Please select the Caller:</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <widget class="QTableView" name="tv_lookupTable">
                <property name="editTriggers">
                 <set>QAbstractItemView::NoEditTriggers</set>
                </property>
                <property name="selectionMode">
                 <enum>QAbstractItemView::SingleSelection</enum>
                </property>
                <property name="selectionBehavior">
                 <enum>QAbstractItemView::SelectRows</enum>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <widget class="QGroupBox" name="gB_ticketData">
             <property name="title">
              <string>Ticket</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_5">
              <item>
               <widget class="QLabel" name="lb_title">
                <property name="text">
                 <string>Title</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="le_ticketTitle"/>
              </item>
              <item>
               <widget class="QLabel" name="lb_description">
                <property name="text">
                 <string>Description</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPlainTextEdit" name="pte_ticketDescription"/>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox">
             <property name="title">
              <string>Ticket actions</string>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QPushButton" name="pb_save">
                <property name="text">
                 <string>Create Ticket</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>StepWidget</class>
   <extends>QWidget</extends>
   <header>H:\Entwicklung\BG_AssignmentAndMaintenanceSystem-AMS\src\system\CustomQT\Stepwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
